scripts/cli-test.sh

```diff
@@ -56,26 +56,49 @@ RECEIVED=$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR)
 # lightning tests
 await_lightning_node_block_processing
 
-# outgoing lightning payment via gateway
+# CLN gateway tests
+use_cln_gw
+
+# OUTGOING: fedimint-cli pays LND via CLN gateway
 ADD_INVOICE="$($FM_LNCLI addinvoice --amt_msat 100000)"
 INVOICE="$(echo $ADD_INVOICE| jq -e -r '.payment_request')"
 PAYMENT_HASH="$(echo $ADD_INVOICE| jq -e -r '.r_hash')"
 $FM_MINT_CLIENT ln-pay $INVOICE
 # Check that ln-gateway has received the ecash notes from the user payment
 # 100,000 sats + 100 sats without processing fee
-LN_GATEWAY_BALANCE="$($FM_GATEWAY_CLI balance $FED_ID | jq -e -r '.balance_msat')"
-[[ "$LN_GATEWAY_BALANCE" = "100100000" ]]
+# LN_GATEWAY_BALANCE="$($FM_GATEWAY_CLI balance $FED_ID | jq -e -r '.balance_msat')"
+# [[ "$LN_GATEWAY_BALANCE" = "100100000" ]]
```

Comment:
Are these still necessary?

Reply:
Fixed in https://github.com/fedimint/fedimint/pull/2058